src/app/utils/testing.utils.ts

```diff
@@ -336,5 +358,30 @@ function mockBodyType(args?: Partial<BodyType>): BodyType {
   const mock: BodyType = {
     description: 'the first letter should be capital'
   } as BodyType;
+
+  return { ...mock, ...args };
+}
+
+function mockPSVBrakes(args?: Partial<Brakes>): Brakes {
+  const mock: Brakes = {
```

Comment:
Also limit the brake keys to essential fields capture on the screen. We should split the keys between this file and the spec under test